seed/challenges/html5-and-css.json

```diff
@@ -3775,8 +3775,8 @@
         "Replace the word <code>black</code> in our <code>body</code> element's background-color with its <code>hex code</code> representation, <code>#000000</code> "
       ],
       "tests": [
-        "assert($(\"body\").css(\"background-color\") === \"rgb(0, 0, 0)\", 'message: Give your <code>body</code> element the background-color of black.');",
-        "assert(code.match(/#000000/g) && code.match(/#000000/g).length > 0, 'message: Use the <code>hex code</code> for the color black instead of the word <code>black</code>. For example <code>body { color&#58; #000000; }</code>');"
+        "assert($(\"body\").css(\"background-color\") === \"rgb(0, 0, 0)\" && code.match(/body\\s?{([\\s\\S]*?)}/i).toString().indexOf(\"background\") != -1, 'message: Give your <code>body</code> element the background-color of black.');",
```

Comment:
I had written te test case origenally to fix `Use Hex Code to Color Elements White`.
The default background-color of the page is `#fff` wich would pass even if there was no code written. Thats the reason why i made the test a bit more explicit.

I simple copy pasted my code afterwards and forgot that the others do not need that test.